lmdeploy/serve/async_engine.py

```diff
@@ -55,7 +55,7 @@ def stop_session(self, session_id: int):
                 input_ids,
                 request_output_len=0,
                 sequence_start=False,
-                sequence_end=False,
+                sequence_end=True,
```

Comment:
This is not a bug.
`stop_session` is used to stop generation, but not to destroy its history conversations that are cached in server.
but `sequence_end=True` is used to terminate the session, including its conversations. 
PR #544 adds a new API `end_session` to deal with the issue